src/test/Compilation.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2 <0.9.0;
+
+pragma experimental ABIEncoderV2;
+
+import "../Test.sol";
+
+// Even though this contract is in the `src/` directory, it is a test contract. It simply inherits
```

Comment:
Move this file to `test/`. See the CI suggestion below.

Reply:
I added a few more contracts to this file to test compilation of each abstract contract in forge-std. Is it possible to compile all 4 with a single forge invocation if they live in the test dir? I'm not too familiar with the `--contracts` / other flags for this

Reply:
If you want to measure compilation time for all four, use the suggestion below. To get individual results, create `test/compilation/` and separate the contracts into files. Then have dedicated steps for each one, e.g. `forge build --skip test --contracts test/compilation/CompilationScrpit.sol --use solc:0.8.17 --via-ir`.

Reply:
Nice, thanks. Four separate benchmarks seems useful, just updated that in b024cee